datahub-web-react/src/app/settingsV2/Preferences.tsx

```diff
@@ -157,7 +167,35 @@ export const Preferences = () => {
                         </StyledCard>
                     </>
                 )}
-                {!showSimplifiedHomepageSetting && !isThemeV2Toggleable && (
+                {canManageApplications && (
```

Comment:
in the resolver for updating application settings, we check `canManageFeatures` - let's make these consistent

Reply:
addressed!

Reply:
Thanks for catching that- my comment was premature. It is now addressed.